### ISSUE TITLE
New-DbaDatabase - Containment

### ISSUE THREAD

**Issue by ksl28** [dataplat/dbatools#9218]
### Summarize Functionality

New-DbaDatabase supports specifying the recovery model when creating the database, it would be nice if it also supported setting the containment type from the beginning.

Today i have to first create the database, and then alter it afterwards using Invoke-Dbaquery

### Is there a command that is similiar or close to what you are looking for?

Yes

**Comment by andreasjordan** [comment]
I will close this now a it has become stale, but can reopen if needed.

**Comment by ksl28** [comment]
Hi @andreasjordan 

I still believe this one is a valid request, so could you please reopen the issue?

**Comment by andreasjordan** [comment]
Ok, I reopend it. But I think this is an edge case and I don't want to invest time in this feature. But maybe someone else might...

**Comment by potatoqualitee** [comment]
i agree this is valid. I will be doing a bunch of dbatools work in a few months and can tackle it at that time 👍🏼
